metis-media-service/src/main/java/eu/europeana/metis/mediaprocessing/RdfBindingFactoryProvider.java

```diff
@@ -7,16 +7,18 @@
 import org.jibx.runtime.JiBXException;
 
 /**
- * This is the super class of all RDF converters, providing an instance of {@link IBindingFactory}.
+ * This class maintains an instance of {@link IBindingFactory} which it can make available upon
+ * request.
  */
-class RdfConverter {
+class RdfBindingFactoryProvider {
```

Comment:
final?

Reply:
True, this class now becomes a utils class with only static methods. I changed it.